### PR TITLE
Add duration chips to CardViews in the Main page

### PR DESCRIPTION
**Pull request by tastelessjolt**
**Adds duration in the UI as small chips at the bottom right of each of the CardView of a Video.**

Please let me know if there are some code-style inconsistencies. 

Screenshot_1660238498

### CODE REVIEW

File: app/src/main/java/ml/bmlzootown/hydravion/card/CardPresenter.kt [hunk 1]
```diff
@@ -132,6 +134,15 @@ class CardPresenter : Presenter() {
                         .into(image)
                 }
 
+                video.metadata?.videoDurationInSecs?.let { totalDurationSecs ->
+                    if (totalDurationSecs <= 0) {
+                        duration.isGone = true
+                    }
+                    else {
+                        duration.text = formatDuration(totalDurationSecs)
```
Comment (NickM-27) [inline app/src/main/java/ml/bmlzootown/hydravion/card/CardPresenter.kt:142]: You will need to set `isVisible = true` here to ensure that the previous state isn't carried over

File: app/src/main/java/ml/bmlzootown/hydravion/card/CardPresenter.kt [hunk 2]
```diff
@@ -166,6 +177,10 @@ class CardPresenter : Presenter() {
             }
         }
 
+        fun formatDuration(durationSecs: Int) : String {
+            return DateUtils.formatElapsedTime(durationSecs.toLong())
+        }
```
Comment (NickM-27) [inline app/src/main/java/ml/bmlzootown/hydravion/card/CardPresenter.kt:182]:
```suggestion
        fun formatDuration(durationSecs: Int) : String =
            DateUtils.formatElapsedTime(durationSecs.toLong())
```

**Review by NickM-27 — CHANGES_REQUESTED** [review]

**Comment by NickM-27** [comment]
Can you include a couple screenshots? Curious to see how it affects items that have tags set? 

Our height is manually set to ensure consistent presentation and want to make sure it stays clean in all cases

**Comment by NickM-27** [comment]
Thanks for the screenshot, that looks great

File: app/src/main/java/ml/bmlzootown/hydravion/card/CardPresenter.kt (hunk 1 above)
Comment (tastelessjolt) [reply app/src/main/java/ml/bmlzootown/hydravion/card/CardPresenter.kt:142]: Oh, you're right! Let me update it now.

**Review by NickM-27 — APPROVED** [review]

**Pull request merged**